sdk/analytics/src/test/java/com/klaviyo/analytics/networking/KlaviyoApiClientTest.kt

```diff
@@ -84,7 +87,20 @@ internal class KlaviyoApiClientTest : BaseTest() {
         mockk<KlaviyoApiRequest>().also {
             every { it.uuid } returns uuid
             every { it.send() } returns status
-            every { it.toJson() } returns "{\"headers\":{\"headerKey\":\"headerValue\"},\"method\":\"GET\",\"query\":{\"queryKey\":\"queryValue\"},\"time\":\"time\",\"uuid\":\"$uuid\",\"url_path\":\"test\"}"
+            every { it.toJson() } returns """
```

Comment:
this is unrelated just something i've been changing when i see it in tests, much easier to read multiline strings